qcli/bsp2svg/converter.py

```diff
@@ -31,52 +31,67 @@ def convert(bsp_file, svg_file, args):
     """
     print(f'Reading {os.path.basename(bsp_file)}')
     bsp_file = Bsp.open(bsp_file)
+    projection_axis = args.projection_axis
+    
+    # Filter faces
+    faces = [face for model in bsp_file.models for face in model.faces]
+    ignore_textures = ['clip', 'hint', 'trigger'] + args.ignore
+    faces = list(filter(lambda f: not (f.texture_name.startswith('sky') or f.texture_name in ignore_textures), faces))
```

Comment:
I don't like the hard coding of the sky texture. That is what the `--ignore` arg is for.

Reply:
it was hardcoded in the first place, in the original code at line 74. I just moved the code and used a lambda to make it more concise. the reason for moving is because the faces need to be filtered _before_ calculating the min / max for the drawing dimensions

Reply:
So I did hard code it myself. 🤦‍♂️

Reply:
it works pretty well though 😄 and I think that the `sky` as part of the texture is actually defined in the game logic to be treated as a special texture. it's fine as it is